source/dem/read_mesh.cc

```diff
@@ -2,11 +2,12 @@
 
 template <int dim, int spacedim>
 void
-read_mesh(const Parameters::Mesh &      mesh_params,
-          const bool &                  restart,
-          const ConditionalOStream &    pcout,
-          Triangulation<dim, spacedim> &triangulation,
-          double &                      triangulation_cell_diameter)
+read_mesh(const Parameters::Mesh &       mesh_params,
+          const bool &                   restart,
+          const ConditionalOStream &     pcout,
+          Triangulation<dim, spacedim> & triangulation,
+          double &                       triangulation_cell_diameter,
+          Parameters::Lagrangian::BCDEM &bc_params)
```

Comment:
I kinda hate the fact that this routine is seperate from the grid one in the core of the librarie. It would be worth it to eventually look into reconciling the two.

Reply:
I agree, I will work on that

Reply:
... will open an issue ;)